fixtures/button.fixture.js

```diff
@@ -0,0 +1,34 @@
+import Button from "@khanacademy/wonder-blocks-button";
+
+import {describe, expect, test, render, simulate} from "./helpers.js";
+
+describe("Button", () => {
+    test("primary", async () => {
+        const element = await render(<Button>Hello, world!</Button>);
+        await expect(element).toMatchScreenshot();
+        await simulate({type: "mousemove", clientX: 50, clientY: 25});
+        await expect(element).toMatchScreenshot({outset: 4});
+        await simulate({type: "mousedown"});
+        await expect(element).toMatchScreenshot();
```

Comment:
We should probably have a way to pass labels to `toMatchScreenshot()` so that it's easier to identify the screenshots.